### ISSUE TITLE
Pack UDP blocks in Simulink and generate a bus output

### ISSUE THREAD

**Issue by vibhoraggarwal** [vibhoraggarwal/inter-process-communication-IPC#8]
Task definition:
1. Currently the [Simulink model with the UDP](https://github.com/vibhoraggarwal/inter-process-communication-IPC/blob/master/usingUDP/udpsendrecdata.slx) sends/receive the data using the already open ports. 
2. We want a Simulink model/block (using the already existing model) than can store and output the data in the form of a Bus, rather than just sending it directly to another port 

Basically, this is how the UDP model would feature:
![UDPcommunicationmodel](https://user-images.githubusercontent.com/19661841/65692576-1635db80-e073-11e9-91a7-b06e2b06e017.jpg)

**Comment by vibhoraggarwal** [comment]
the simulink model, and the code for receiving a array of numbers is updated [here](https://github.com/vibhoraggarwal/inter-process-communication-IPC/tree/master/usingUDP)
